js/react_native/android/src/main/java/ai/onnxruntime/reactnative/OnnxruntimeModule.java

```diff
@@ -169,59 +162,70 @@ public WritableMap loadModel(String uri, InputStream modelStream, ReadableMap op
   public WritableMap run(String key, ReadableMap input, ReadableArray output, ReadableMap options) throws Exception {
     OrtSession ortSession = sessionMap.get(key);
     if (ortSession == null) {
-      throw new Exception("Model is not loaded " + key);
+      throw new Exception("Model is not loaded: " + key);
     }
 
     RunOptions runOptions = parseRunOptions(options);
 
     long startTime = System.currentTimeMillis();
     Map<String, OnnxTensor> feed = new HashMap<>();
     Iterator<String> iterator = ortSession.getInputNames().iterator();
-    while (iterator.hasNext()) {
-      String inputName = iterator.next();
-
-      ReadableMap inputMap = input.getMap(inputName);
-      if (inputMap == null) {
-        throw new Exception("Can't find input: " + inputName);
+    try {
```

Comment:
there is not too much changes in this block. I just added a try-finally to wrap the code and a few lines of comments.